codechecker_common/plist_parser.py

```diff
@@ -97,7 +97,7 @@ def parse(self, fileobj):
         return self.root
 
 
-def read_plist(path):
+def load_plist(plist_file):
```

Comment:
@gyorb The argument of this function is not a plist file. It is the content of a plist file.
Also this function will not load the given plist file, it will only parse it.

Reply:
Actually it is a file object and not the file content, I've made some renaming to make it more explicit.